WordPress/gutenberg
# Site Editor: add fullscreen close button

## Description


Add Fullscreen mode close but to the Site Editor, similar to the one we already have in post editor.

Follow up of Follow up of https://github.com/WordPress/gutenberg/pull/20691.

## How has this been tested?



1. Open the experimental Site Editor.
2. Toggle the fullscreen mode with `wp.data.dispatch( 'core/edit-site' ).toggleFeature( 'fullscreenMode' );`.
3. Verify that the button only appears when the fullscreen mode is active.
4. Verify that clicking on it takes you back to wp-admin dashboard.

## Screenshots 

<img width="290" alt="Screenshot 2020-03-20 at 22 05 26" src="https://user-images.githubusercontent.com/1182160/77206320-f3065900-6af6-11ea-8c5b-5b9f0bca3f47.png">


## Types of changes


New feature (non-breaking change which adds functionality)


## Checklist:
